### DISCUSSION TITLE
Complex Table/Cells

### DISCUSSION THREAD

**Discussion by wolfassi123** [deepdoctection/deepdoctection#360] [General]
I was wondering if any of you managed to solve some of the issues that we face when dealing with a complex table in a pdf. For example several subheaders, cells that span several columns or rows (they can span several columns or rows, but their text would span only through 1 cell/row)

Was trying to find a solution to complex table using several libraries but so far always came a bit too short.
The library does pretty good at simplifying some of the basic tasks, but still a complex table seems to have no fix yet.

Below is an example table that I made on Excel inspired by a real case scenario that I have faced. Is there a way to fix the fact that some cells/headers span several columns for example. The only detection we have that would allow us to know for how many columns a header span is basically the horizontal line at the bottom of the header.

![fixed_test_complex_table_image](https://github.com/deepdoctection/deepdoctection/assets/82727504/238444a7-c138-4653-8a9b-498e90d59558)

**Comment by AndrejPer** [comment]
Good question, I'd be interested too.

**Comment by DemonSwans** [comment]
Me with my team (I'm not connected to author of deepdoctection) are trying to improve table detection and structure recognition by using different methods we are still looking for table detection but for structure recognition we are trying to include UniTable into dd pipeline
